web/package.json

```diff
@@ -53,6 +54,7 @@
     "react-reflex": "4.0.9",
     "react-router-dom": "6.2.1",
     "react-scripts": "5.0.1",
+    "react-sortable-hoc": "2.0.0",
```

Comment:
This lib has a warning in their repo:

This library is no longer actively maintained. It will continue to receive critical security updates, but there are no new features planned. In future versions of React, the [findDOMNode](https://reactjs.org/docs/react-dom.html#finddomnode) method will be deprecated. This method is a critical piece of the architecture of react-sortable-hoc, and the library will stop working in the future when that method is removed from react-dom.

All development efforts have been redirected towards [@dnd-kit](https://github.com/clauderic/dnd-kit). It provides feature parity, built with a modern and extensible architecture, supports complex use-cases and has accessibility features built-in. New consumers are strongly encouraged to adopt @dnd-kit instead of adopting react-sortable-hoc.

Does it worth it to move to `dnd-kit`?